pkg/kubernetes/kube_test.go

```diff
@@ -108,7 +115,37 @@ func TestPostitiveResourceOperation(t *testing.T) {
 	err = kc.ResourceOperation(OperationDelete, fileName)
 	g.Expect(err).ShouldNot(gomega.HaveOccurred())
 }
+func TestPostitiveMultipleResourcesOperation(t *testing.T) {
+	var (
+		err               error
+		g                 = gomega.NewWithT(t)
+		dynScheme         = runtime.NewScheme()
+		fakeDynamicClient = fakeDynamic.NewSimpleDynamicClient(dynScheme)
+		fakeDiscovery     = fakeDiscovery.FakeDiscovery{}
+	)
+	kc := Client{
+		DynamicInterface:   fakeDynamicClient,
+		DiscoveryInterface: &fakeDiscovery,
+		FilesPath:          "../../test/templates",
+	}
+	const fileName = "test-multi-resourcefile.yaml"
+
+	testResourceList, err := multipleResourcesFromYaml(fileName)
+	if err != nil {
+		t.Errorf("Failed getting the test resource from the file %v: %v", fileName, err)
+	}
+	for _, kubernetesResource := range testResourceList {
+		testResource := kubernetesResource.Resource
+		fakeDiscovery.Fake = &fakeDynamicClient.Fake
+		fakeDiscovery.Resources = append(fakeDiscovery.Resources, newTestAPIResourceList(testResource.GetAPIVersion(), testResource.GetName(), testResource.GetKind()))
+
+		err = kc.ResourceOperation(OperationCreate, fileName)
```

Comment:
The point of this Unit Test would be to test `MultiResourceOperation` not `ResourceOperation`. So you need to call the former here.